General/OOD/SemanticConstructorOverloading/SemanticConstructorOverloading.Problem/Enum Based Constructor Parameters/Good/SampleClass.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SemanticConstructorOverloading.Problem.Enum_Based_Constructor_Parameters.Good
+{
+    /// <summary>
+    /// ClassEnums contains enumerations for AValues and BValues which obviate the need for string parameters
+    /// default handling and validation, as the enumerations are both wholly constrained and the default value
+    /// for any enumeration is 0.
```

Comment:
Overall problem comment sounds redundant. It could be more precise.

Also, it should refer to a problem. Use <seeref> tag for it and explain how it solves the problem and not why is it good in general.